Tests/NewsServiceTests/Utils/NewsPublisherFactoryTests.cs

```diff
@@ -1,8 +1,36 @@
-﻿
+﻿using Interfaces;
+using NewsService;
+using NewsService.Utils;
+using NUnit.Framework;
+using CentralBankIntegration;
+using System;
 
 namespace NewsServiceTests.Utils
 {
     class NewsPublisherFactoryTests
     {
+        private NewsPublisherTypes centralBankNewsPublisherType = NewsPublisherTypes.CentralBank;
+        private INewsPublisher centralBankNewsPublisher;
+
+        [SetUp]
+        public void Initialize()
+        {
+            centralBankNewsPublisher = new CentralBankNewsPublisher();
+        }
+
+        [Test]
+        public void CreateNotNullNewsPublisher()
+        {
+            foreach (NewsPublisherTypes type in Enum.GetValues(typeof(NewsPublisherTypes)))
```

Comment:
`var type`

Reply:
No, it doesn't understand var, because it enum

Reply:
Oh, ok, didn't know